src/core/Aggregators.java

```diff
@@ -34,7 +33,7 @@
     LERP,   /* Regular linear interpolation */
     ZIM,    /* Returns 0 when a data point is missing */
     MAX,    /* Returns the <type>.MaxValue when a data point is missing */
-    MIN     /* Returns the <type>.MinValue when a data point is missing */
+    MIN,    /* Returns the <type>.MinValue when a data point is missing */
```

Comment:
@johann8384 @manolama Hey， I just found that "first" and "last" aggregators already exist in "next" branch, and the implementation code makes me a little confused...